### ISSUE TITLE
control events in non-measured notation?

### ISSUE THREAD

**Issue by axgeertinger** [music-encoding/music-encoding#601]
The MEI 3 model.controleventLike has changed somewhat into model.controlEventLike.cmn in MEI 4. It seems that its members are no longer available in non-measured notation (within `<layer>`, for instance, where they used to be allowed) except inside editorial markup. Is that intentional? Specifically, my question is: Where can I put a fermata in square (neume) notation?

**Comment by ahankinson** [comment]
I think the short answer is that fermata weren't really considered as part of the neume notation module, and are only really available in CMN. I'm not sure that 'mixed mode' CMN/Neumes were tested.

You could probably create your own customization that re-enables the CMN module with the neumes module. Or are you already using the `mei-all.rng`?

**Comment by axgeertinger** [comment]
Yes, I'm using `mei-all.rng`. The problem is that the members of model.controlEventLike.cmn are not allowed in `<layer>` as they used to be in MEI 3. Would it be a problem to reinstate that option? I'd prefer not having to customize the schema. Below is an example from what we are encoding. Fermatas in lines 2 and 5.
![jesp30](https://user-images.githubusercontent.com/3779074/59676778-9624ed80-91c8-11e9-974d-1edd0bf71109.png)

**Comment by bwbohl** [comment]
For cases like this we should definitely come up with a solution, which might mean to reinstate the control events in layer in mei all and disallowing them in CMN?

**Comment by davidrizo** [comment]
In mensural it happens the same situation. I suppose they were removed because the `tstamp` attribute was difficult to use in non measure notation. However, not it seems we can use this kind of notation that can be used to reinstate the control events:

`<fermata staff="1" startid="#note1" place="above" />`

**Comment by rettinghaus** [comment]
`<fermatas>` are currently part of `model.controlEventLike.cmn` 
First step is to make them part of `model.controlEventLike`
Also it may be in `model.sectionLike` to make it available in a `<section>` 
That way it would be possible to use them in unmeasured music and also to use it *between* two measures without trying to figure out a logical value for `@tstamp` to place it on top of a bar line.

**Comment by kepper** [comment]
might be related to #322. Probably a bigger issue…
